bin/release-build.sh

```diff
@@ -12,5 +12,10 @@ if [[ $# != 1 ]]; then
 fi
 
 log_file="/build/logs/${1}.log"
-nohup time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir ${1%%-*} > $log_file 2>&1 &
+copy_dir="${1%%-*}"
+
+nohup sh -c \
+        "echo time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir $copy_dir \
+         && time ruby ${BUILD_DIR}/scripts/vmpublish.rb --copy-dir $copy_dir" \
```

Comment:
1. Yes, although in `vmpublish.rb` I could loop over the publish destination and not the image type.

2. I propose to remove it from there :)

3. Agreed that we don't want to publish if parts of the build is broken. I was referring to the fact that the easiest way to do the upload is to add it in the inner build loop in `vmbuild.rb` before we rename the image. That would mean we interleave things: build X - upload X - build Y, upload Y.  We could put the publish loop of course completely at the end of `vmbuild.rb`.

The strongest argument for separating the build and publish phases is 2 - so that you can do some sanity testing before you publish something to a potentially large number of users. What do you think about that?